tests/SprykerEcoTest/Zed/Payone/Business/Payment/PaymentLogTest.php

```diff
@@ -35,7 +35,7 @@ public function testGetPaymentLogs()
         $apiLogs[] = $this->createPayoneApiLog(PayoneApiConstants::REQUEST_TYPE_CAPTURE, PayoneApiConstants::RESPONSE_TYPE_TIMEOUT);
 
         $orderCollection = (new OrderCollectionTransfer())
-            ->addOrders($this->orderTransfer);
+            ->addOrder($this->orderTransfer);
```

Comment:
When did this change? Looks like a BC break.

Reply:
This was released in the previous major. For now, I've only fixed the tests that had been broken in the previous release. "PaymentLogTest.php" - It touches the only test file